Assets/Scripts/API/BlocksFile.cs

```diff
@@ -584,8 +584,7 @@ private void ReadRmbFldHeader(BinaryReader reader, int block)
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown3 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Unknown4 = reader.ReadUInt32();
                 blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].FactionId = reader.ReadUInt16();
-                blocks[block].DFBlock.RmbBlock.FldHeader.BuildingDataList[i].Sector = reader.ReadInt16();
```

Comment:
Thank you. :) I've ended up using a different building ID setup in Daggerfall Unity than Daggerfall uses, and I'd like to keep the two systems distinct (rather than have two things called buildingID that don't equate to each other).

If the value is encountered elsewhere, just read as 2x Int16 values like now (Sector/LocationID). Cheers mate.